File: 2271-rearrange-array-elements-by-sign/rearrange-array-elements-by-sign.py
class Solution(object):
    def rearrangeArray(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        po=[]
        ne=[]
        for i in nums:
            if(i>0):
                po.append(i)
            else:
                ne.append(i)
        for i in range(len(nums)//2):
            nums[2*i]=po[i]
            nums[2*i+1]=ne[i]
        return nums

File: 2271-rearrange-array-elements-by-sign/rearrange-array-elements-by-sign.cpp
class Solution {
public:
    vector<int> rearrangeArray(vector<int>& nums) {
        vector<int> pos;
        vector<int> neg;
        vector<int> ans;
        for(auto it: nums){
            if(it>0){
                pos.push_back(it);
            }else{
                neg.push_back(it);
            }
        }
        int n=(nums.size()/2);
        for(int i=0; i <n; i++){
            ans.push_back(pos[i]);
            ans.push_back(neg[i]);
        }
        return ans;

        
    }
};